docs/pages/versions/unversioned/sdk/local-authentication.md

```diff
@@ -58,10 +64,10 @@ Attempts to authenticate via Fingerprint/TouchID (or FaceID if available on the
 #### Arguments
 
 - **options (_object_)** -- An object of options.
-  - **promptMessage (_string_)** -- A message that is shown alongside the TouchID or FaceID prompt. (**iOS only**)
-  - **cancelLabel (_string_)** -- Allows to customize the default `Cancel` label shown. (**iOS only**)
+  - **promptMessage (_string_)** -- A message that is shown alongside the TouchID, FaceID or Biometric prompt.
```

Comment:
I removed the iOS only tags here because this is also implemented in Android now (via #8219)